libs/utils/env.py

```diff
@@ -707,6 +707,18 @@ def _init_platform(self):
         # Adding topology information
         self.platform['topology'] = self.topology.get_level("cluster")
 
+        # Adding kernel build information
+        kver = self.target.kernel_version
+        self.platform['kernel'] = {t: getattr(kver, t, '') \
```

Comment:
- Nit: don't need the `\`.
- Can't we just leave out the keys that `KernelVersion` didn't find, instead of having an empty string?